src/components/cards/employees-card/EmployeeCard.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="employee-card">
+
+      <h4 class="employeeCardName">{{employeeFirstName}}</h4>
+      <h4 class="employeeCardLastName">{{employeeLastName}}</h4>
+      <p class="employeeCardRole">{{employeeRole}}</p>
+
+  </div>
+</template>
+
+<script>
+export default {
+    name: 'employee-card',
+    props: {
+    employeeFirstName: { type: String, default: 'Quentin' },
```

Comment:
- Naming convention: should be `firstName`
- default value should be an empty string